images/base/files/usr/local/bin/mount-product-files

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+bundle=$(sed -r 's/.*,"bundle":"([/0-9a-z\.]*)",.*/\1/')
```

Comment:
this code needs comments, in particular this first part.
- what are we doing here?
- what format is the input expected to be that we are regexing?

Reply:
is it safe to assume the json will be on a single line? can we link to the spec?
is it safe to assume there will not be any other key matching this regex at a different part of the data?

Reply:
The [runtime spec](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md), even if the json is multiline I doubt the are going to pass the `bundle` attribute with the key at one line and the value at other, also looks like there is no more `"bundle"` value there, a part from that CI will fail misserably since this affects all the containers.